server/eb/app.py

```diff
@@ -159,12 +160,13 @@ def get_csp_hashes(app, app_config):
         logging.critical("No config file found")
         sys.exit(1)
 
+    # TODO: looks deprecated - remove
     dataroot = os.getenv("CXG_DATAROOT")
     if dataroot:
         logging.info("Configuration from CXG_DATAROOT")
         app_config.update_server_config(multi_dataset__dataroot=dataroot)
 
-    # overwrite configuration for the eb app
+    # TODO: this doesn't make sense
```

Comment:
I think we only support cxg matrices in the explorer at this point?

Reply:
I will leave it just in case.